.travis.yml

```diff
@@ -1,27 +1,25 @@
+sudo: required
```

Comment:
Sure. The problem is it takes time and would do that for every ruby version. I am referring to installing it _just_ when launching the JRuby test.